lib/src/main/java/com/group3/racingbot/inventory/filter/FilterManager.java

```diff
@@ -0,0 +1,186 @@
+/**
+ * 
+ */
+package com.group3.racingbot.inventory.filter;
+
+import java.util.ArrayList;
+
+import com.group3.racingbot.inventory.Iterator;
+
+/**
+ * Used to store and chain iterator decorators which act as filters together to generate an output.
+ * @author Nick Sabia
+ */
+public class FilterManager<T> {
+	private ArrayList<IteratorDecorator<T>> filters;
```

Comment:
For class instance variable unless absolutely have to should use Collections interface rather than concrete class for the type.